lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -12,7 +12,7 @@ import { usePlaceholderDocumentTitleInDev } from '../../utils/hooks';
 import AssignmentActivity from './AssignmentActivity';
 import CourseActivity from './CourseActivity';
 import DashboardFooter from './DashboardFooter';
-import OrganizationActivity from './OrganizationActivity';
+import HomeActivity from './HomeActivity';
```

Comment:
Perhaps `AllCoursesActivity` would be a clearer name that matches up with the UI.

Reply:
I considered that, but that would introduce an inconsistency with how the other activity components are named.

* `CourseActivity`: Displays information about one course, and lists its assignments.
* `AssignmentActivity`: Displays information about one assignment, and lists its students.

This approach makes it more future proof for when other entities can be listed in every activity section.

When that happens, the `HomeActivity` will support listing courses, assignments and students, so naming it `AllCoursesActivity` will be incorrect.

So I suggest we name it by the scope, not by what it currently lists. Perhaps `Home` is not the best name though.

Reply:
> I considered that, but that would introduce an inconsistency with how the other activity components are named.

I don't see the inconsistency. The current naming of each view matches the context for that view:

- For `CourseActivity`, the context is one course
- For `AssignmentActivity`, the context is one assignment
- For `HomeActivity`/`AllCoursesActivity`, the context is "all courses you have access to"

Whether the activity is presented as a list of courses, students or assignments doesn't affect the context.

Reply:
I think having `AllCoursesActivity` and `CourseActivity` is going to be problematic and/or confusing eventually 😅, but I'll change it

Reply:
It could happen that we want to make a conceptual change to what the components represent. In that case we should of course rename them again.